packages/auth/amplify_auth_cognito_dart/lib/src/auth_plugin_impl.dart

```diff
@@ -86,9 +87,14 @@ class AmplifyAuthCognitoDart extends AuthPluginInterface<
     implements Closeable {
   /// {@macro amplify_auth_cognito_dart.amplify_auth_cognito_dart}
   AmplifyAuthCognitoDart({
-    SecureStorageInterface? credentialStorage,
+    SecureStorageFactory? secureStorageFactory,
```

Comment:
Good point.  Depends on how we want to evolve the usage of this factory though.  

I assumed there might be other secure storage instances created, hence the reason we switched to factory.  If not, we can change back to credentialStorage, my only concern is if factory is used to create other instances and now the name becomes misleading.

Reply:
Hm, I see your point. IMO the main benefit of using the factory is to "hide" to scope from the customer, but it could hypothetically be used to create other instances in the future. I think that is unlikely, but I guess the name change is fine.

Reply:
Okay, I will keep the name change as is then.  I'll keep comment open for others to consider.